From b75c8b4978b2dd985459c94abc2d70e1e8f71490 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Thu, 24 May 2012 22:32:07 +0300
Subject: [PATCH 192/370] confd.h: redo the fixes of commit b21ef18

Commit 926088a somehow reverted the fixes made by commit b21ef18.
(Was it because Anatoliy didn't merge (or rebase) prior to pushing?)

This commit re-introduces those fixes:

    confd.h: fix some grammar and style errors

    - Use quotes properly: <a href="X">Y</a>, not <a href=X>"Y"</a>.
    - Fix a couple of typos.
    - Replace `@li' with `-'.
    - Improve wording in several places.
    - Use proper case in enumerations.
---
 conf/confd.h | 40 ++++++++++++++++++++--------------------
 1 file changed, 20 insertions(+), 20 deletions(-)

diff --git a/conf/confd.h b/conf/confd.h
index 51b6721..6c40830 100644
--- a/conf/confd.h
+++ b/conf/confd.h
@@ -52,6 +52,7 @@
  *
  * <hr> <!------------------------------------------------------------>
  * @section confd-fspec-data Data Structures
+ *
  * - c2_reqh_confd_service --- represents configuration service instance
  *   registered in request handler.
  *
@@ -72,7 +73,8 @@
  * configuration requests and replies requires the use of the RPC
  * layer as a transport. Services have to register c2_fop_types for
  * each FOP they support.
- * The following FOPs are defined for confd (onwire.ff):
+ *
+ * The following FOPs are defined for confd (see conf/onwire.ff):
  * - c2_conf_fetch --- configuration request;
  * - c2_conf_fetch_resp --- Confd's response to c2_conf_fetch;
  * - c2_conf_update --- Update request;
@@ -80,29 +82,29 @@
  * - c2_conf_stat --- Statistics request;
  * - c2_conf_stat_resp --- Confd's response to c2_conf_stat.
  *
- * For each FOP, FOP format, FOP operation vector, FOP types, RPC item
- * types have to be defined. The following structures have to be
- * defined for c2_conf_fetch FOP:
+ * FOP format, FOP operation vector, FOP type, and RPC item type have
+ * to be defined for each FOP.  The following structures are defined
+ * for c2_conf_fetch FOP:
  * - struct c2_fop_type_format c2_conf_fetch_tfmt --- defines format
- *   registed in *.ff used in confd;
+ *   registered in *.ff used in confd;
  * - struct c2_fop_type c2_conf_fetch_fopt --- defines FOP type;
  * - struct c2_fop_type_ops c2_conf_fetch_ops --- defines FOP
  *   operation vector;
  * - struct c2_rpc_item_type c2_rpc_item_type_fetch --- defines RPC
- *   item type for given fop.
+ *   item type.
  *
- * FOPs used in configuration service are defined in @ref confd_fop_dfspec .
+ * FOPs used in configuration service are defined in @ref confd_fop_dfspec.
  *
  * <hr> <!------------------------------------------------------------>
  * @section confd-fspec-sub Subroutines
  *
  * - c2_confd_service_register()  - registers confd service in the system.
- * - c2_confd_service_unregister() - unregisters confd service in the system.
+ * - c2_confd_service_unregister() - unregisters confd service.
  *
  * <!------------------------------------------------------------------>
  * @subsection confd-fspec-sub-setup Initialization and termination
  *
- * To parametrize confd, user should provide the following arguments:
+ * To parameterize confd, user should provide the following arguments:
  * through environment variables.  See @ref confd-fspec-cli.
  *
  * Confd is initiated and put into operation by request handler logic,
@@ -112,21 +114,19 @@
  *
  * Initial configuration database is manually created prior to startup
  * with @ref yaml2db utility. Confd assumes that:
- *  - Configuration db is created before confd started;
- *  - Confd uses configuration caching schema, described in
- *    <a href=https://docs.google.com/a/xyratex.com/document/d/1JmsVBV8B4R-FrrYyJC_kX2ibzC1F-yTHEdrm3-FLQYk/view>
- *    "High level design of Colibri’s configuration database schema"</a>
- *    document.
+ *  - configuration db is created before confd started;
+ *  - the schema of configuration database conforms to
+ *    <a href="https://docs.google.com/a/xyratex.com/document/d/1JmsVBV8B4R-FrrYyJC_kX2ibzC1F-yTHEdrm3-FLQYk/view">
+ *    HLD of Colibri’s configuration database schema"</a>.
  *
  *  - Format of configuration db, represented in yaml can be found in @ref yaml2db.
  *
  * The following errors may occur while using the configuration db:
- * @li Db is empty or is in an unrecognized format.
- * @li Db layout is different from that mentioned in
- *     <a href=https://docs.google.com/a/xyratex.com/document/d/1JmsVBV8B4R-FrrYyJC_kX2ibzC1F-yTHEdrm3-FLQYk/view>
- *     "High level design of Colibri’s configuration database schema"</a>
- *
- * @li Key is not found.
+ * - db is empty or is in an unrecognized format;
+ * - db schema does not conform to
+ *   <a href="https://docs.google.com/a/xyratex.com/document/d/1JmsVBV8B4R-FrrYyJC_kX2ibzC1F-yTHEdrm3-FLQYk/view">
+ *   HLD of Colibri’s configuration database schema</a>;
+ * - key is not found.
  *
  * While initialization process, confd has open configuration db and
  * fill internal cache of configuration objects with their
-- 
1.8.3.2

